libs/sdk/assembly/crypto.ts

```diff
@@ -294,31 +344,36 @@ class CryptoECDSA {
         return null
     }
 
-    static importKey(keyName: string, keyPair: SimpleKeyPair, format: string = 'raw', extractable: boolean = false): KeyECC | null {
-
-        if (keyPair.privateKey && keyPair.publicKey)
-            return null;
-
-        const keyBuff = keyPair.privateKey ? keyPair.privateKey : keyPair.publicKey;
+    static importKey(keyName: string, keyData: string, algorithm: string = 'ecc256', format: string = 'raw', extractable: boolean = false): KeyECC | null {
```

Comment:
This is also a breaking change in userland this time. This would require a major version bump to ship.

Reply:
This one will have to stay. It is not breaking the retro compatibility of previously built and deployed app. However, as pointed out, it will break sdk interface, therefore, it makes sense to bump the sem versioning.